[2025-08-15T00:00:03.954+0000] {processor.py:161} INFO - Started process (PID=39889) to work on /opt/airflow/dags/scripts/check_minio_file.py
[2025-08-15T00:00:03.956+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scripts/check_minio_file.py for tasks to queue
[2025-08-15T00:00:03.961+0000] {logging_mixin.py:188} INFO - [2025-08-15T00:00:03.958+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/check_minio_file.py
[2025-08-15T00:00:03.982+0000] {logging_mixin.py:188} INFO - [2025-08-15T00:00:03.976+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/scripts/check_minio_file.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/check_minio_file.py", line 13, in <module>
    from minio import Minio
ModuleNotFoundError: No module named 'minio'
[2025-08-15T00:00:03.983+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/check_minio_file.py
[2025-08-15T00:00:04.067+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scripts/check_minio_file.py took 0.130 seconds
[2025-08-15T00:00:14.710+0000] {processor.py:161} INFO - Started process (PID=39899) to work on /opt/airflow/dags/scripts/check_minio_file.py
[2025-08-15T00:00:14.714+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scripts/check_minio_file.py for tasks to queue
[2025-08-15T00:00:14.716+0000] {logging_mixin.py:188} INFO - [2025-08-15T00:00:14.716+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/check_minio_file.py
[2025-08-15T00:00:14.732+0000] {logging_mixin.py:188} INFO - [2025-08-15T00:00:14.727+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/scripts/check_minio_file.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/check_minio_file.py", line 13, in <module>
    from minio import Minio
ModuleNotFoundError: No module named 'minio'
[2025-08-15T00:00:14.732+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/check_minio_file.py
[2025-08-15T00:00:14.801+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scripts/check_minio_file.py took 0.109 seconds
[2025-08-15T00:00:25.909+0000] {processor.py:161} INFO - Started process (PID=39909) to work on /opt/airflow/dags/scripts/check_minio_file.py
[2025-08-15T00:00:25.992+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scripts/check_minio_file.py for tasks to queue
[2025-08-15T00:00:26.215+0000] {logging_mixin.py:188} INFO - [2025-08-15T00:00:26.190+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/check_minio_file.py
[2025-08-15T00:00:26.835+0000] {logging_mixin.py:188} INFO - [2025-08-15T00:00:26.806+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/scripts/check_minio_file.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/check_minio_file.py", line 13, in <module>
    from minio import Minio
ModuleNotFoundError: No module named 'minio'
[2025-08-15T00:00:26.843+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/check_minio_file.py
[2025-08-15T00:00:27.188+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scripts/check_minio_file.py took 1.947 seconds
[2025-08-15T00:00:37.504+0000] {processor.py:161} INFO - Started process (PID=39919) to work on /opt/airflow/dags/scripts/check_minio_file.py
[2025-08-15T00:00:37.508+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scripts/check_minio_file.py for tasks to queue
[2025-08-15T00:00:37.512+0000] {logging_mixin.py:188} INFO - [2025-08-15T00:00:37.510+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/check_minio_file.py
[2025-08-15T00:00:37.528+0000] {logging_mixin.py:188} INFO - [2025-08-15T00:00:37.526+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/scripts/check_minio_file.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/check_minio_file.py", line 13, in <module>
    from minio import Minio
ModuleNotFoundError: No module named 'minio'
[2025-08-15T00:00:37.529+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/check_minio_file.py
[2025-08-15T00:00:37.606+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scripts/check_minio_file.py took 0.117 seconds
[2025-08-15T00:00:48.818+0000] {processor.py:161} INFO - Started process (PID=39929) to work on /opt/airflow/dags/scripts/check_minio_file.py
[2025-08-15T00:00:48.826+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scripts/check_minio_file.py for tasks to queue
[2025-08-15T00:00:48.838+0000] {logging_mixin.py:188} INFO - [2025-08-15T00:00:48.832+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/check_minio_file.py
[2025-08-15T00:00:48.866+0000] {logging_mixin.py:188} INFO - [2025-08-15T00:00:48.861+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/scripts/check_minio_file.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/check_minio_file.py", line 13, in <module>
    from minio import Minio
ModuleNotFoundError: No module named 'minio'
[2025-08-15T00:00:48.867+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/check_minio_file.py
[2025-08-15T00:00:48.961+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scripts/check_minio_file.py took 0.188 seconds
